sdcm/tester.py

```diff
@@ -2712,8 +2712,14 @@ def run_fstrim_on_all_db_nodes(self):
         This function will run fstrim command all db nodes in the cluster to clear any bad state of the disks.
         :return:
         """
+        # if used ebs volumes with aws backend fstrim is not supported
+        # fstrim: /var/lib/scylla: the discard operation is not supported
+        if self.db_cluster.is_ebs_volumes_attached():
+            self.log.info("fstrim is not supported on aws ebs volumes")
+            return
+
         for node in self.db_cluster.nodes:
-            node.fstrim_scylla_disks()
+            node.remoter.sudo('fstrim -v /var/lib/scylla')
```

Comment:
You need to address it on AWSNode level, meaning override fstrim_scylla_disks there:

```
class AWSNode:
....
    def fstrim_scylla_disks(self):
        if self.parent_cluster.is_ebs_volumes_attached():
            self.log.info("fstrim is not supported on aws ebs volumes")
            return
        return super().fstrim_scylla_disks()
...
  
```

Reply:
Or even better, to move out run_fstrim_on_all_db_nodes to BaseScyllaCluster and address it there

Reply:
done